altlayer-test-node.bash

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
```

Comment:
How hard would it be to not duplicate this bash script and the docker compose file?

Reply:
Shouldnt be hard, I can do that!

Reply:
https://github.com/EspressoSystems/nitro-testnode/pull/59/commits/b41364dfef60af181ab8c13a5bfe43e4e2ec7390